.travis.yml

```diff
@@ -115,6 +115,12 @@ script:
     if [[ -v BUILD_PRESTO_DOCKER ]]; then
       docker/build-local.sh
     fi
+  - |
+    if [[ -v MAVEN_CHECKS ]]; then
+      # TODO enable this for presto-parser
+      env MAVEN_OPTS="${MAVEN_INSTALL_OPTS}" ./mvnw clean compile test-compile -DskipTests -B -T C1 -P errorprone-compiler-presto \
+        -pl '!presto-server-rpm,!presto-server,!presto-docs,!presto-thrift-api,!presto-parser'
```

Comment:
What is wrong with thrift-api and parser?

Reply:
it doesn't build with errorprone.